ecpy/measure/editors/execution_editor/editor.enaml

```diff
@@ -125,10 +125,14 @@ enamldef _BaseExecutionEditor(Container):
     #: Reference to the model keeping track of the pools currently used.
     attr _model
 
-    padding = 0
+    padding = 1
```

Comment:
The padding is the white space around the widgets that the container add. I could revert to 0, at one point I though this was causing an issue but it was not so.